### PR TITLE
Patch over circular dep between Box3 and Sphere

### PR DESCRIPTION
**Pull request by chazcb**
Box3 relies on Sphere, and Sphere relies on Box3 to instantiate an instance of Box3 and use that instance's methods inside `setFromPoints`. This is a fairly naive patch to resolve the circular dependency so that we can load three.js source files using webpack / babel.

### CODE REVIEW

File: src/math/Sphere.js [hunk 1]
```diff
@@ -28,10 +28,12 @@ Sphere.prototype = {
 
 	setFromPoints: function () {
 
-		var box = new Box3();
+		var box;
 
 		return function setFromPoints( points, optionalCenter ) {
 
+			box = box || new Box3();
```
Comment (Mugen87) [inline src/math/Sphere.js]: Maybe you can add here a comment like `box = box || new Box3(); // see #10547`

Comment (mrdoob) [inline src/math/Sphere.js]: I would also do...

```js
if ( box === undefined ) box = new Box3();
```

**Comment by chazcb** [comment]
Thanks for the suggestions @mrdoob and @Mugen87!

File: src/math/Sphere.js (hunk 1 above)
Comment (WestLangley) [inline src/math/Sphere.js]: We prevent circular references all the time using this pattern. Why the inline comment in this case?

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**